sktime/annotation/datagen.py

```diff
@@ -8,25 +8,139 @@
 from sklearn.utils.validation import check_random_state
 
 
+def multivariate_mean_shift(
+    means: npt.ArrayLike,
+    lengths: npt.ArrayLike,
+    vars: Union[npt.ArrayLike, float] = 1.0,
```

Comment:
FYI `vars` is a reserved keyword in python https://docs.python.org/3/library/functions.html#vars, would be worth changing it to avoid clashes